src/components/Dialog.tsx

```diff
@@ -115,14 +126,82 @@ export const Modal = styled.div<{ color: Color }>`
   flex-direction: column;
   height: 100%;
   left: 0;
-  overflow: hidden;
```

Comment:
just checking - removing this doesn't break the entrance/exit animation for modals? you can test with the token selection modal (focus on the corners)

Reply:
great catch, fixed by adding border radius to hidden wrapper